models/src/test/scala/coop/rchain/models/InterpolateSpec.scala

```diff
@@ -23,6 +23,26 @@ class InterpolateSpec extends FlatSpec with TripleEqualsSupport with Matchers {
     assert(interpolated === send)
   }
 
+  it should "replace data sent" in {
+    val fixedElement: Par = GInt(10)
+    val send              = Send(GPrivateBuilder(), Seq(GString("#key1"), GString("#key2"), fixedElement))
```

Comment:
Honestly I'd prefer those tests inputs to be expressed in rholang, but won't push on that :)

Reply:
Since the `Interpolate` works on `Par`s I wanted to keep the test strictly in `Par` and the examples were straightforward enough that it wasn't a problem.